src/map/clif.c

```diff
@@ -10727,7 +10727,7 @@ static void clif_parse_LoadEndAck(int fd, struct map_session_data *sd)
 			}
 		}
 
-		if(sd->pd && sd->pd->pet.intimate > 900)
+		if(sd->pd != NULL && sd->pd->pet.intimate >= PET_INTIMACY_LOYAL)
 			clif->pet_emotion(sd->pd,(sd->pd->pet.class_ - 100)*100 + 50 + pet->hungry_val(sd->pd));
```

Comment:
```suggestion
			clif->pet_emotion(sd->pd, (sd->pd->pet.class_ - 100) * 100 + 50 + pet->hungry_val(sd->pd));
```

Reply:
Actually you're right, but when I start applying code style here, I'd have to do it for the whole 350+ lines of that function.
This seems to be a little bit too much effort just for changing a value in an if-condition. Don't you agree?